src/msg.c

```diff
@@ -546,6 +546,9 @@ static int8_t msg_auth(struct session *session,
 		return result;
 	}
 
+	l_queue_foreach(mydevice->schema, schema_dup_foreach,
```

Comment:
This code only handle devices that have schema in device_id_list.

Reply:
After we receive schema we should duplicate from session->schema_list to device_id_list too.
Because new devices registered are added to device_id_list and don't have schema

Reply:
Ok